src/main/java/collections/CollectionWithoutDuplicates.java

```diff
@@ -0,0 +1,30 @@
+package collections;
+
+import java.util.Arrays;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class CollectionWithoutDuplicates {
+    public static void main(String[] args) {
+
+        String[] myStrings = {"string1", "string2", "string3", "string1", "string5"};
+        Integer[] myNumbers = {1, 2, 3, 1, 5};
+
+        System.out.println("\nArray of Strings objects:");
+        System.out.println(Arrays.toString(myStrings));
+        System.out.println("\nSet of Strings (without duplicates):");
+        System.out.println(removeDuplicates(myStrings));
+
+        System.out.println("\nArray of Integers objects: ");
+        System.out.println(Arrays.toString(myNumbers));
+        System.out.println("\nSet of Integer (without duplicates):");
+        System.out.println(removeDuplicates(myNumbers));
+
+    }
+
+    public static <V> Set<V> removeDuplicates(V[] aCollection) {
+        Set<V> clearedSet = Arrays.stream(aCollection).collect(Collectors.toSet());
```

Comment:
there is no need to create a separate variable and return it on a new line. also, according to the task, you should have a Collection (Collection <T> aCollection) as a method's argument, not an array - this will simplify the solution as well (you can just create set from some collection)

Reply:
done

Reply:
return on separate line was fixed, however, there is still an array (V[] aCollection) as a method's argument, should be Collection<T> collection

Reply:
done

Reply:
done

Reply:
nice! please change 'V' to 'T', which refers to Type parameter name - https://docs.oracle.com/javase/tutorial/java/generics/types.html
also, please provide the parametrized type for this generic in method's argument - should be "public static <T> Collection<T> removeDuplicates(Collection<T> aCollection)"

Reply:
done.
  As for <V> and <T> it is caused by task's text where "V" mentioned, which confused me.

Reply:
good catch, seems that we need to fix it as well!